server/memphis_handlers_tags.go

```diff
@@ -154,165 +148,254 @@ func AddTagsToEntity(tags []models.CreateTag, entity_type string, entity_id prim
 func DeleteTagsByStation(id primitive.ObjectID) {
```

Comment:
change to deleteTagsFromStation and I think it not has to start with capital letter.
Check all function inside this file and change where needed